### PR TITLE
Iter5

### PR DESCRIPTION
**Pull request by herou3**
Added logic for system var

### CODE REVIEW

File: cmd/shortener/main.go [hunk 1]
```diff
@@ -1,3 +1,50 @@
+// пакеты исполняемых приложений должны называться main
 package main
 
-func main() {}
+import (
+	"flag"
```
Comment (arttet) [inline cmd/shortener/main.go:5]: Please sort imports
https://stackoverflow.com/questions/38704769/golang-import-grouping-by-package

File: internal/services/context/context.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+func (us *UrlsStorage) CreateShortURL(fullURL string) (string, error) {
+	isNotFound := true
+	shortURLId := ""
+	if fullURL == "" {
+		return "", fmt.Errorf("fullURL is empty %s", errFullURLIsEmpty)
+	}
+	for isNotFound {
+		shortURLId = tools.String(8)
```
Comment (arttet) [inline internal/services/context/context.go]: Please rename this one
```
tools.String(8)
```

File: internal/services/handlers/get/get.go [hunk 3]
```diff
@@ -0,0 +1,26 @@
+package get
+
+import (
+	"github.com/herou3/url-shortener/internal/services/context"
+	"net/http"
+)
+
+var uh = context.GetUH()
```
Comment (arttet) [inline internal/services/handlers/get/get.go]: Please don't use global variables

**Review by arttet — CHANGES_REQUESTED** [review]
Please add golangci-lint https://github.com/golangci/golangci-lint
Please add a code coverage report https://about.codecov.io/blog/getting-started-with-code-coverage-for-golang/
Please add gofmt action https://github.com/marketplace/actions/check-code-formatting-using-gofmt

File: internal/services/context/context.go (hunk 2 above)
Comment (herou3) [reply internal/services/context/context.go]: Changed name

File: internal/server/server.go [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+	"github.com/herou3/url-shortener/internal/services/handlers/get"
+)
+
+// Server is a server with all the batteries included
+type Server struct {
+	Mux *chi.Mux
+}
+
+// Init returns new server instance
+func Init() *Server {
```
Comment (arttet) [inline internal/server/server.go:15]: Please rename this function

File: internal/services/handlers/create/create.go [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+import (
+	"io"
+	"net/http"
+	"regexp"
+
+	"github.com/herou3/url-shortener/internal/services/context"
+)
+
+func HandleCreateShortURL(response http.ResponseWriter, request *http.Request) {
```
Comment (arttet) [inline internal/services/handlers/create/create.go:11]:
```suggestion
func CreateShortURLHandler(response http.ResponseWriter, request *http.Request) {
```

File: internal/services/handlers/create/create.go [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+	if request.Method != http.MethodPost {
+		response.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	fullURL, _ := io.ReadAll(request.Body)
+	isMatch, _ := regexp.MatchString("((http|https)://)(www.)?[a-zA-Z0-9@:%._+~#?&/=]{2,256}\\.[a-z]{2,6}\\b([-a-zA-Z0-9@:%._+~#?&/=]*)", string(fullURL))
```
Comment (arttet) [inline internal/services/handlers/create/create.go:18]: It would be better if you add a comment about this reg ex.

**Review by arttet — APPROVED** [review]
Please add golangci-lint https://github.com/golangci/golangci-lint
Please add gofmt action https://github.com/marketplace/actions/check-code-formatting-using-gofmt

**Pull request closed without merging**